packages/gcf-utils/package.json

```diff
@@ -53,4 +54,4 @@
     "package.json",
     "LICENSE"
   ]
-}
+}
```

Comment:
Let's make the angry red squiggle go away

Reply:
I actually don't see an angry red squiggle anywhere 😬